bookstore/handlers.py

```diff
@@ -21,15 +21,7 @@ class BookstoreVersionHandler(APIHandler):
 
     @web.authenticated
     def get(self):
-        self.finish(
-            json.dumps(
-                {
-                    "bookstore": True,
-                    "version": version,
-                    "bookstore_validation": self.settings['bookstore_validation'],
-                }
-            )
-        )
+        self.finish(json.dumps({"bookstore": True, **self.settings['bookstore']}))
```

Comment:
~~We should still expose the version on the API endpoint for our own debugging. Or wait... does `self.settings['bookstore']` have this?~~ I see, that's below.

Is there anything in `self.settings['bookstore']` that shouldn't be leaked out as part of the payload? One code smell I see here is that if anyone adds a new property to the settings, it's not going to be obvious that it will get leaked out onto the API endpoint here. I think this should still use specific keys.

Reply:
@stormpython I think e54de64 addresses @rgbkrk's concerns and so as long as it looks good to you I think this PR is good to go.